syntax = "proto3";

package timezone;

service TimezoneConverter {
    // Converts the time into a target timezone
    rpc ConvertTime(TimeConversionRequest) returns (TimeConversionResponse){}
}

message TimeConversionRequest{
    string time_str = 1; //exemplo "2025-07-11 23:10:10"
    string source_timezone = 2; //exemplo "America/Sao_Paulo"
    string target_timezone = 3; //exemplo "America/Bogota" 
}

message